src/super_gradients/training/datasets/detection_datasets/detection_dataset.py

```diff
@@ -98,6 +94,7 @@ def __init__(
                                         differ based on transforms.
         :param max_num_samples:         If not None, set the maximum size of the dataset by only indexing the first n annotations/images.
         :param cache:                   Whether to cache images or not.
+        :param cache_annotations:       Whether to cache annotations or not.
```

Comment:
Can you please add what exactly it means when `cache_annotations` is set to True and False?
E.g why one would prefer to enable of disable ones.

Reply:
Is that clear enough? 59ee4b74e41a1ea69aad2a6157fb06e9f7f83406